src/components/menu/menu.module.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../palette.scss";
+
+.menuWrapper {
+  height: 62px;
+  align-items: center;
+  border-bottom: 1px solid $gray;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.menuItems {
+  display: flex;
+}
+.menuItem {
+  height: 61px;
+  align-items: center;
+  display: flex;
+  margin-right: 30px;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 20px;
+  text-transform: uppercase;
+  color: $white;
+  cursor: pointer;
+  border-bottom: 1px solid transparent;
+}
+.menuItem:hover, .active {
```

Comment:
You can use SCSS features and declare CSS classes directly in main class declaration with usage of `&`, check it out [here](https://sass-scss.ru/documentation/sassscript/&_v_sassscript/)